.github/workflows/config/spelling_allowlist.txt

```diff
@@ -132,7 +135,10 @@ canonicalization
 canonicalize
 canonicalizer
 canonicalizes
+chol
+ci
 codebase
+coeffs
```

Comment:
I don't think these are words and therefore should probably not be added to the allowlist.